features/expectations/method.feature

```diff
@@ -1,11 +1,11 @@
-@stable
+@javascript @stable
 Feature: Method
 
   Background:
     Given you expect HTTP message method "POST"
 
   Scenario: HTTP message method match
-    When real method is "POST"
+    When real HTTP message method is "POST"
```

Comment:
Will it ever be response? Maybe we can be specific here with using "request" instead of "HTTP message"

Reply:
We can refine vocabulary used in `gavel-spec` feature definitions as a separate task.

Here I've updated it in order to correspond to the Cucumber step definitions in `gavel.js`, which expects an exact wording (previously "HTTP message" phrase was missing, and the step has been skipped).